modules/local/suppa_clusterevents.nf

```diff
@@ -0,0 +1,54 @@
+process CLUSTEREVENTS {
+    tag "${cond1}-${cond2}"
+    label 'process_high'
+    stageInMode = 'copy'
```

Comment:
why?

Reply:
This is an edge-case with SUPPA - the program internally cannot recognise symlinks so we have to copy the file into the execution folder.